app/build.gradle

```diff
@@ -80,15 +80,31 @@ dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$rootProject.coroutinesVersion"
 
     // Compose
-    implementation "androidx.compose.runtime:runtime:$rootProject.composeVersion"
-    implementation "androidx.compose.ui:ui:$rootProject.composeVersion"
-    implementation "androidx.compose.foundation:foundation:$rootProject.composeVersion"
-    implementation "androidx.compose.foundation:foundation-layout:$rootProject.composeVersion"
-    implementation "androidx.compose.material:material:$rootProject.composeVersion"
-    implementation "androidx.compose.ui:ui-viewbinding:$rootProject.composeVersion"
-    implementation "androidx.ui:ui-tooling:$rootProject.composeVersion"
-    implementation "androidx.compose.runtime:runtime-livedata:$rootProject.composeVersion"
-    implementation "com.google.android.material:compose-theme-adapter:$rootProject.composeVersion"
+    implementation("androidx.compose.runtime:runtime:$rootProject.composeVersion") {
+        version { strictly '1.0.0-SNAPSHOT' }
```

Comment:
You shouldn't need these changes with the proposed changes below

Reply:
Works! Thanks